src/injected/drawing-controller.ts

```diff
@@ -112,27 +112,26 @@ export class DrawingController {
         await drawer.drawLayout();
     };
 
-    private enableVisualizationInIFrames = async (
+    private enableVisualizationInIFrames = (
         frame: HTMLIFrameElement,
         frameResults: AssessmentVisualizationInstance[],
         configId: string,
-    ): Promise<void> => {
+    ): void => {
         const message: VisualizationWindowMessage = {
             elementResults: frameResults,
             isEnabled: true,
             featureFlagStoreData: this.featureFlagStoreData,
             configId: configId,
         };
 
-        await this.frameMessenger.sendMessageToFrame(
-            frame,
-            this.createFrameRequestMessage(message),
-        );
+        // This message recursively triggers enableVisualization in all iframes.
+        // Intentionally floating this promise.
+        void this.frameMessenger.sendMessageToFrame(frame, this.createFrameRequestMessage(message));
```

Comment:
I tried some changes locally to avoid having to do this - my results are in branch [dbjorge:no-floating-promises-3-b](https://github.com/pownkel/accessibility-insights-web/compare/no-floating-promises-3...dbjorge:no-floating-promises-3-b?expand=1)

Reply:
Thanks for investigating this, I merged in the changes from your branch.